src/components/comment-list-item.component.js

```diff
@@ -3,79 +3,79 @@
 
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { StyleSheet, View, Text, TouchableOpacity, Image } from 'react-native';
+import styled from 'styled-components';
 import { GithubHtmlView } from 'components';
 import { Icon } from 'react-native-elements';
 import ActionSheet from 'react-native-actionsheet';
 
 import { translate, relativeTimeToNow } from 'utils';
 import { colors, fonts, normalize } from 'config';
 
-const styles = StyleSheet.create({
-  container: {
-    paddingRight: 10,
-    paddingTop: 10,
-    backgroundColor: 'transparent',
-  },
-  header: {
-    flexDirection: 'row',
-    marginLeft: 10,
-    alignItems: 'center',
-  },
-  avatarContainer: {
-    backgroundColor: colors.greyLight,
-    borderRadius: 17,
-    width: 34,
-    height: 34,
-  },
-  avatar: {
-    width: 34,
-    height: 34,
-    borderRadius: 17,
-  },
-  titleSubtitleContainer: {
-    justifyContent: 'center',
-    flex: 1,
-    marginLeft: 10,
-  },
-  dateContainer: {
-    flex: 0.15,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-  linkDescription: {
-    ...fonts.fontPrimaryBold,
-    fontSize: normalize(13),
-    color: colors.primaryDark,
-  },
-  date: {
-    color: colors.greyDark,
-  },
-  commentContainer: {
-    paddingTop: 5,
-    marginLeft: 54,
-    borderBottomColor: colors.greyLight,
-    borderBottomWidth: 1,
-  },
-  commentBottomPadding: {
-    paddingBottom: 15,
-  },
-  commentText: {
-    fontSize: normalize(12),
-    color: colors.primaryDark,
-  },
-  commentTextNone: {
-    ...fonts.fontPrimary,
-    color: colors.primaryDark,
-    fontStyle: 'italic',
-  },
-  actionButtonIconContainer: {
-    paddingTop: 5,
-    paddingBottom: 10,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-});
+const Container = styled.View`
+  padding: 10px 10px 0 0;
```

Comment:
This seems to cause a Travis build failure, could you have a look?

Reply:
@machour It is due to my stylelint regular expression. Fixed now.